src/views/Home/components/LostAndFoundCard/components/FoundReports/index.tsx

```diff
@@ -15,17 +16,30 @@ const formatDate = (date: string) => {
 const FoundReports = () => {
   const [foundReports, setFoundReports] = useState<MissingItemReport[] | null>(null);
   const [snackbar, setSnackbar] = useState({ message: '', severity: undefined, open: false });
+  const [user, setUser] = useState({ AD_Username: '' });
   const navigate = useNavigate();
   const isMobile = useMediaQuery('(max-width:375px)');
 
   const createSnackbar = useCallback((message, severity) => {
     setSnackbar({ message, severity, open: true });
   }, []);
 
+  useEffect(() => {
+    const fetchUserData = async () => {
```

Comment:
Same thing here, use useUser instead if possible (to avoid excessive API requests)